backend/.env.example

```
@@ -27,6 +27,12 @@ AUCTION_CONTRACT_ID=C... # Auction mechanism contract address
 
+TYPEORM_SYNCHRONIZE=true
+PORT=3000
+
+# Logging Configuration
+LOG_LEVEL=info
+
 # --- Keeper Service (Automation) ---
 KEEPER_ENABLED=false # Toggle for background automation tasks
-KEEPER_SECRET_KEY= # Private key used by the keeper to sign transactions
\ No newline at end of file
+KEEPER_SECRET_KEY= # Private key used by the keeper to sign transactions
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)